libs/util/shared/src/lib/links/link-utils.ts

```diff
@@ -94,6 +94,33 @@ export const FORMATS = {
     color: '#de630b',
     mimeTypes: ['application/x-dxf', 'image/x-dxf'],
   },
+  html: {
+    extensions: ['html', 'htm'],
+    priority: 12,
+    color: '#f2bb3a',
+    mimeTypes: ['text/html'],
+  },
+  fgb: {
```

Comment:
these keys are also used for display I think, so maybe going with "flatgeobuf" would make it clearer :)

Reply:
Okay ! :) But won't it put "flatgeobuf" as a format in the url? i tested and `https://demo.ldproxy.net/zoomstack/collections/airports/items?f=flatgeobuf` returns an error, while  `https://demo.ldproxy.net/zoomstack/collections/airports/items?f=fgb` is alright (its a download though, but it will be relevant for the other ticket).

But maybe my understanding of this function is wrong! I understand that the key is used as the format name.

Reply:
hmm, maybe that approach is wrong then; ogc-client offers a "getCollectionItemsUrl" function, this function could be responsible for taking in a mime type and generting a correct url out of it, instead of the application being responsible for appending a "f=" format param itself

Reply:
Yes, but in this case, the whole logic of the api-form would have to be changed :grimacing: The format, limit and offset being added dynamically when the input values change. Would it mean that the baseUrl would now come from getCollectionItemsUrl? Wouldn't it be easier to have a `displayName` for each mimeType, instead of using the key?

Reply:
Then I guess we can stay with fgb for now; in principle I would say the url generation should be the responsibility of ogc-client here, but this can be done later